### ISSUE TITLE
Cxbx's interactions with 1050Ti Mobile/Dell 7567 Laptop and JSRF

### ISSUE THREAD

**Issue by Dent115** [Cxbx-Reloaded/Cxbx-Reloaded#1325]
Multiple issues (screenshots and dumps can be added if requested I'm new here and not so tech savvy):
- Blackscreen with audio during for JSRF when set to anything but default resolution with 1050ti as display adapter
- speech letters out of place and scaled to default resolution when using Igpu as display adapter
- black screen when boost is used on Igpu 
- black screen after loading on igpu if not set to default resolution 
- GPU not detected unless d3d8tod9 is put into files
- texture issues (white buildings ) when resolution is raised (occurs in intro too and then crashes) 
basically the game forces me into default resolution and hands me a fat L if i ever dare to even raise it one resolution size higher.

**Comment by ergo720** [comment]
Wrong place to post this, put it here: https://github.com/Cxbx-Reloaded/game-compatibility/issues/703
